docs/internals/contributing/new-contributors.txt

```diff
@@ -30,8 +31,8 @@ Start with these steps to discover Django's development process.
   behavior, even if you don't fix the bug itself. See more at
   :ref:`how-can-i-help-with-triaging`
 
-* **Look for tickets that are accepted and review patches to build familiarity
-  with the codebase and the process**
+Look for tickets that are accepted and review patches to build familiarity with the codebase and the process
+------------------------------------------------------------------------------------------------------------
 
   Mark the appropriate flags if a patch needs docs or tests. Look through the
   changes a patch makes, and keep an eye out for syntax that is incompatible
```

Comment:
I think this is too long for a header, so let's move some of this into the description
To get you started, something like:
```suggestion
Review patches of accepted tickets
----------------------------------

This will help you to build familiarity with the codebase and processes. Mark
the appropriate flags if a patch needs docs or tests. Look through the changes
a patch makes, and keep an eye out for syntax that is incompatible with older
```

Reply:
Thanks for the suggestion. I didn't even think to make any content changes.